README.md

```diff
@@ -380,7 +340,11 @@ You can provide an error message for a single validation with `.withMessage()`.
 req.assert('email', 'Invalid email')
     .notEmpty().withMessage('Email is required')
     .isEmail();
-var errors = req.validationErrors();
+
+req.getValidationErrors(function(errors){
```

Comment:
Your method doesn't accept a callback, so please remove this example or convert it to promise style